docs/manual/manual.adoc

```diff
@@ -14,7 +14,6 @@
 :scap:  http://scap.nist.gov/
 :nist:  http://www.nist.gov/
 :cpe:   https://cpe.mitre.org/
-:cce:   https://cce.mitre.org/
```

Comment:
we might still want to include this link because we use CCEs, we just don't use the API we have written for them